### ISSUE TITLE
horse riding (stealing/riding the same horse)

### ISSUE THREAD

**Issue by AngryTanuki** [tiltedphoques/bug-feature-reports#37]
default: skyrim together reborn w/address library requirement

we were trying to not teleport as we noticed that caused desync in enemies. we had been separated for about 10 minutes. after meeting up with my friend in ivarstead we started making our way up the mountain for the greybeard quest, at this time my horse was not there for my friend i was floating. as we neared the greybeards fort the horse became visible for my friend. my friend decided to "steal" my horse while i was on it. this caused my character to deform on my screen and he took control of the horse. it was all fun and laughter for me till we reached the doors of high hrothgar where my friend and i both got off the horse. this sent me flying across the map.
Related formIDs|
---------------|
[horse] unsure of exact id

## Steps to reproduce

How to reproduce this issue. Be as specific as possible and try to only include instructions you believe are necessary to reproduce said issue, i.e.

1. player A riding horse (unsure if stolen matters)
2. player B starts riding the same horse
3. both players dismount the horse in relative same time(not sure if if same time is required)
4. Player A gets flung

## Reproduction rate

Please check just _one_ of the following reproduction rates:

Did not test unsure was a first time event

- [ ] Consistent: I can reproduce this bug by following the steps to reproduce 100% of the time (*every time*)
- [ ] Occasional: I can reproduce this bug by following the steps to reproduce around 50% of the time
- [ ] Rare: I can reproduce this bug by following the steps to reproduce around 10% or less of the time

## Expected result

deformed character for one or more players, a player being tossed across the map

## Evidence (optional)

My POV:    https://www.youtube.com/watch?v=Rh9XxEs2-Rk
Friends POV:     https://www.youtube.com/watch?v=xoS0DAXaw-s&t=1s

**Comment by AngryTanuki** [comment]
misunderstood expected result however most any result was not expected maybe a crash or getting stuck, its a horse meant for 1 person.

**Comment by androbourne** [comment]
I can confirm this matters no matter if the horse is stolen or not.  I purchased my own horse and team mates can "steal" it from me and I can steal it back. Team mates also can't purchase their own horse.

We had another issue with the headless horsemen. I can stop him to talk to the horsemen as the host but my team mates only have option to "steal" the headless horsemens horse. Funny to see but its broken for sure.

**Comment by AngryTanuki** [comment]
Yes can confirm 100% we tested it with a player owned horse the vid was just great cause we happened to be at high hrothgar when I got tossed across the map
